core_lib/src/managers/viewmanager.cpp

```diff
@@ -181,7 +181,7 @@ void ViewManager::rotate(float degree)
 {
     if (mCurrentCamera)
     {
-        mCurrentCamera->rotate(degree);
+        mCurrentCamera->rotate(static_cast<qreal>(degree));
```

Comment:
Given that the camera uses qreal for rotation, we should probably use that as the parameter for this function. Likewise for `ViewManager::rotation`, scaling, etc.